From e1685c08c31a90efa11777975b254bcba196eb58 Mon Sep 17 00:00:00 2001
From: Volker Krause <vkrause@kde.org>
Date: Tue, 2 Mar 2021 17:27:19 +0100
Subject: [PATCH] Update weather API endpoint

Version 1.9 has been discontinued, but 2.0/classic returns the same
format as far as our use-case is concerned.
---
 src/weather/weatherforecastmanager.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/weather/weatherforecastmanager.cpp b/src/weather/weatherforecastmanager.cpp
index cac846e..0224470 100644
--- a/src/weather/weatherforecastmanager.cpp
+++ b/src/weather/weatherforecastmanager.cpp
@@ -180,7 +180,7 @@ void WeatherForecastManager::fetchNext()
     QUrl url;
     url.setScheme(QStringLiteral("https"));
     url.setHost(QStringLiteral("api.met.no"));
-    url.setPath(QStringLiteral("/weatherapi/locationforecast/1.9/"));
+    url.setPath(QStringLiteral("/weatherapi/locationforecast/2.0/classic"));
     QUrlQuery query;
     query.addQueryItem(QStringLiteral("lat"), QString::number(tile.latitude()));
     query.addQueryItem(QStringLiteral("lon"), QString::number(tile.longitude()));
-- 
GitLab

